docs/edgeql/parameters.rst

```diff
@@ -150,11 +148,54 @@ When using a client library, pass the idiomatic null pointer for your language:
     select <required str>$name;
 
 
+Default parameter values
+------------------------
+
+When using optional parameters, you may want to provide a default value to use
```

Comment:
Maybe add a link to here at this point? https://www.edgedb.com/docs/edgeql/sets#coalescing Seeing a sudden ?? can be jarring at time for those new to the operator.

(But not the word 'coalescing' in this page as having a new keyword in the middle of everything can be confusing)

Reply:
I like the idea of linking. I'm not sure how to do it without saying "coalescing" though. Do you have a suggestion? Does it help if I give a brief explanation inline as well as using the term and linking it?

Reply:
Maybe just a link at the word default? Personally my mind just sees the word coalescing and turns it to the word default. I guess coalescing is originally a SQL term so makes sense to name it that way though I originally seized on a similar Rust pattern when learning ?? that includes the word default:

https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or_default

(Though it's not an exact equivalent, actually more similar to a method called or https://doc.rust-lang.org/std/option/enum.Option.html#method.or) https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=10e2f8fe3c3311b2177992b45d2931e3

Reply:
I'm afraid it might be weird for the word "default" to link to coalescing documentation. I feel like coalesce is a tool that can be used to create a default value, but the two are not synonymous. I'm afraid it could be especially confusing for a low-vision user using a screen reader if they just had it read the links and didn't get the rest of the context.

I do like the idea of a link though. Let me see what I can do with it…

Reply:
![CleanShot 2023-09-22 at 08 19 29@2x](https://github.com/edgedb/edgedb/assets/2027711/1a8bf529-daaf-44b2-8c27-3fbb03a4656a)
I think this is a decent compromise. I refer to is as the `??` operator but also call it by its name in the parenthetical. I think it's important that we do call it by name so that users will know how to talk about it if they need to ask a question or share how they accomplished something with another user. This may be the only context they see it in, so I think it's actually a good idea to use the name here as well.

It was definitely deficient without this link though. I gave no context, so I'm glad you called it out!